server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/FilterTest.java

```diff
@@ -79,6 +79,7 @@ public void setup() throws Throwable {
         RestAssured.requestSpecification = jmapRequestSpecBuilder
             .setPort(jmapServer.getProbe(JmapGuiceProbe.class).getJmapPort())
             .build();
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
```

Comment:
Not really found of this has it leads to a lot of logs when calling _await_ method.